XenAdmin/Wizards/ExportWizard/ExportSelectVMsPage.cs

```diff
@@ -263,6 +263,22 @@ private bool CheckSpaceRequirements(out string errorMsg)
 			return true;
 		}
 
+	    private bool CheckDiskSizeForTransfer(out string errorMsg)
+	    {
+	        errorMsg = string.Empty;
+
+	        foreach (VM vm in VMsToExport)
+	        {
+	            if (GetTotalVmSize(vm) > 2 * Util.BINARY_TERA && !ExportAsXva)

```

Comment:
I would check `!ExportAsXva` first in order to avoid the size calculation if the user has not selected ovf/ova.